layouts/in.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="lc-page-wrapper with-sliding-menu">
+    <tool-bars :disableError="getIfDisableError" />
+
+    <sliding-menu />
+
+    <header class="lc-page-header">
+      <div class="lc-container-0">
+        <div class="lc-container-1">
+          <div class="underlay gray lc-mobile-hide" />
+          <div class="lc-container-2">
+            <site-header/>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <div class="lc-page-content">
+
+      <div class="lc-container-1">
+        <div class="underlay gray lc-mobile-hide" />
+        <div class="lc-container-2 lc-padding-bottom-8">
+          <user-info-form />
+        </div>
+
+        <!-- BEGIN - Tab bar section -->
+        <md-tabs class="lc-tabs lc-container-2 lc-width-2-3 md-transparent" md-sync-route>
+
+          <template slot="md-tab"  slot-scope="{ tab }">
+            <span class="lc-tab-item-label">{{ tab.label }}</span>
+          </template>
+
+          <md-tab
+            id="overview-tab"
+            :md-label="$t('In.tab.overview')"
+            to="/in" />
+          <md-tab
+            id="bonus-tab"
+            :md-label="$t('In.tab.bonus')"
+            to="/in/bonus" />
+          <md-tab
+            id="history-tab"
+            :md-label="$t('In.tab.history')"
+            to="/in/bonus/history" />
```

Comment:
does `to` support vue-router syntax like ` { name: 'in-bonus' }`? prefer `name` if so

Reply:
Not support